examples/prediction_upload/video_predictions.ipynb

```diff
@@ -64,18 +64,7 @@
         "!pip install -q 'labelbox[data]'"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    set_feature_schema_id(features_schema, bbox_with_radio_subclass_annotation_ndjson)
</pre><p>Do we need these steps, if we can reference the feature schemas by name?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>